x-pack/plugin/rollup/src/main/java/org/elasticsearch/xpack/downsample/TransportRollupIndexerAction.java

```diff
@@ -50,6 +55,7 @@ public class TransportRollupIndexerAction extends TransportBroadcastAction<
     private final Client client;
     private final ClusterService clusterService;
     private final IndicesService indicesService;
+    private final TransportCancelTasksAction cancelTasksAction;
```

Comment:
Maybe use the `client` field here instead of adding `cancelTasksAction` here? The cancel task action is available in `ClusterAdminClient`.

Reply:
I think the `cancelTasksAction` field can be removed here and just use `client` here to invoke the cancel action.